nemo_skills/dataset/wmt24pp/prepare.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+from pathlib import Path
+
+from langcodes import Language
+
+from datasets import load_dataset
+
+LANG2CODE = {"de": "de_DE", "es": "es_MX", "fr": "fr_FR", "it": "it_IT", "ja": "ja_JP"}
```

Comment:
Can we just directly use the long code instead of adding this conversion dict? Because this will limit the utility of this script to just the 5 languages above, and if we use two-letter code we will have extendability problem once we include zh_CN and zh_TW.